bundles/FileExplorerBundle/src/FileExplorerBundle.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Pimcore\Bundle\FileExplorerBundle;
+
+use Pimcore\Extension\Bundle\AbstractPimcoreBundle;
+use Pimcore\Extension\Bundle\Traits\PackageVersionTrait;
+use Pimcore\Bundle\FileExplorerBundle\DependencyInjection\FileExplorerExtension;
+use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
+
+class FileExplorerBundle extends AbstractPimcoreBundle
+{
+
+    use PackageVersionTrait;
+
+    public function getComposerPackageName(): string
```

Comment:
I guess this is for future references right? 
Are there plans to move it to its own repo or will it be kept under bundles?

Reply:
Yes! File Explorer is supposed to be a separate composer package.